### ISSUE TITLE
Confused by multiply n to both clf_logits and clf_losses?

### ISSUE THREAD

**Issue by Vimos** [huggingface/pytorch-openai-transformer-lm#39]
I am confused by the code below. 

https://github.com/huggingface/pytorch-openai-transformer-lm/blob/eafc28abdfadfa0732f03a0fc65805c5bfb2ffe7/train.py#L52

https://github.com/huggingface/pytorch-openai-transformer-lm/blob/eafc28abdfadfa0732f03a0fc65805c5bfb2ffe7/train.py#L54

Is this due to any normalization? Thanks!

**Comment by thomwolf** [comment]
Yes, I don't understand that myself either. It was in the original TensorFlow code so I kept it as is. I have removed it in a fork of this code that I use for another project and it doesn't really change the results. I may try to dig further one day.

**Comment by pranoy-k** [comment]
Hi Thomas,

Do you have any insights now for this issue.

**Comment by thomwolf** [comment]
I don't
